sessions/js-functions/pizza-analyzer/js/index.js

```diff
@@ -0,0 +1,29 @@
+console.clear();
+
+const pizzaInput1 = document.querySelector('[data-js="pizza-input-1"]');
+const pizza1 = document.querySelector('[data-js="pizza-1"]');
+const pizzaInput2 = document.querySelector('[data-js="pizza-input-2"]');
+const pizza2 = document.querySelector('[data-js="pizza-2"]');
+const outputSection = document.querySelector('[data-js="output-section"]');
+
+const output = document.querySelector('[data-js="output"]');
+
+pizzaInput1.addEventListener("input", () => {
+  // write your code here
+});
+
+pizzaInput2.addEventListener("input", () => {
+  // write your code here
+});
+
+function calculatePizzaGain(diameter1, diameter2) {
+  // write your code here
+}
+
+function updatePizzaDisplay(pizzaElement, newSize) {
+  // write your code here
```

Comment:
should we comment that this function belongs to task 2? and the others respectively?